requirements.txt

```diff
@@ -1,2 +1,3 @@
 future
 requests
+pytz
```

Comment:
Done in https://github.com/twingly/twingly-search-api-python/commit/13bef577aea8f11528f1d6e8b7b929b835c0fae4